docs/semgrep-ci/overview.md

```diff
@@ -19,11 +20,13 @@ import MoreHelp from "/src/components/MoreHelp"
 Semgrep CI runs fully in your build environment: **your code is never sent anywhere**.
 :::
 
-## Getting started
+**Note**: Semgrep CI is a subcommand of Semgrep CLI, so you can test the code and behavior of Semgrep CI in your command line using `semgrep ci` command. This means that all the capabilities of Semgrep CLI are available in Semgrep CI.
```

Comment:
Tried to summarize the note as you suggested. Not sure if the first sentence is entirely true (about testing it). Feel free to suggest anything you would like to see here, please. This should be a friendly introduction to our CI, it does not have to be very technical, it should just give an image of possibilities. @brendongo @underyx

Reply:
```suggestion
**Note**: Semgrep CI is a subcommand of Semgrep CLI, so you can test the code and behavior of Semgrep CI on your machine by running `semgrep ci`.
```
I tried to remove some mentions of the word 'command'. I also removed the 'capabilities of Semgrep CLI' sentence which sounded very vague to me.